python/tools/prepare_build_environment.sh

```diff
@@ -38,18 +40,6 @@ make -j2 install
 cd ..
 rm -r build-release
 
-cd python
-for PYTHON_VERSION in cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39
-do
-    PYTHON_ROOT=/opt/python/$PYTHON_VERSION
-    $PYTHON_ROOT/bin/pip install -r install_requirements.txt
-    $PYTHON_ROOT/bin/python setup.py bdist_wheel
-    rm -rf build
-done
-
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/intel/oneapi/dnnl/latest/cpu_gomp/lib
-for wheel in dist/*
-do
-    auditwheel show $wheel
-    auditwheel repair $wheel
-done
+# Download test data
+curl -o transliteration-aren-all.tar.gz https://opennmt-models.s3.amazonaws.com/transliteration-aren-all.tar.gz
+tar xf transliteration-aren-all.tar.gz -C tests/data/models/
```

Comment:
Can this be done elsewhere? Looks like this should happen in `CIBW_BEFORE_TEST`.

Reply:
Sure, I can do that. My intention was to avoid downloading the same model for every Python version, but the model is just ~44MB anyway.